### ISSUE TITLE
KeyError: 'process.absent' for salt 2015.5.3 version

### ISSUE THREAD

**Issue by ThomasZhou** [saltstack/salt#26316]
I have a simple sls file:
```saltstack
stop_nodecellar:
  process.absent:
    - name: nodejs
    - user: root
```
When execute it, return a exception
```python-traceback
[root@zt-centos states]# salt '*' state.sls nodecellar/webapp/nodecellar-stop
server01:
    The minion function caused an exception: Traceback (most recent call last):
      File "/usr/local/lib/python2.7/dist-packages/salt/minion.py", line 1161, in _thread_return
        return_data = func(*args, **kwargs)
      File "/usr/local/lib/python2.7/dist-packages/salt/modules/state.py", line 681, in sls
        ret = st_.state.call_high(high_)
      File "/usr/local/lib/python2.7/dist-packages/salt/state.py", line 2067, in call_high
        ret = dict(list(disabled.items()) + list(self.call_chunks(chunks).items()))
      File "/usr/local/lib/python2.7/dist-packages/salt/state.py", line 1623, in call_chunks
        running = self.call_chunk(low, running, chunks)
      File "/usr/local/lib/python2.7/dist-packages/salt/state.py", line 1769, in call_chunk
        self._mod_init(low)
      File "/usr/local/lib/python2.7/dist-packages/salt/state.py", line 612, in _mod_init
        self.states['{0}.{1}'.format(low['state'], low['fun'])]  # pylint: disable=W0106
      File "/usr/local/lib/python2.7/dist-packages/salt/utils/lazy.py", line 90, in __getitem__
        raise KeyError(key)
    KeyError: 'process.absent'
```
```yaml
root@zhout:/# salt-call --versions
           Salt: 2015.5.3
         Python: 2.7.6 (default, Mar 22 2014, 22:59:56)
         Jinja2: 2.8
       M2Crypto: 0.21.1
 msgpack-python: 0.4.6
   msgpack-pure: Not Installed
       pycrypto: 2.6.1
        libnacl: Not Installed
         PyYAML: 3.10
          ioflo: Not Installed
          PyZMQ: 14.0.1
           RAET: Not Installed
            ZMQ: 4.0.4
           Mako: 0.9.1
        Tornado: Not Installed
```

**Comment by jfindlay** [comment]
@ThomasZhou, thanks for the report.  The process state module requires the [psutil](https://pypi.python.org/pypi/psutil) package to be installed.  What is the output of `salt '*' sys.doc ps.pkill`?

**Comment by ThomasZhou** [comment]
It's lack of psutil, after I install it, it's run correct

**Comment by jfindlay** [comment]
@ThomasZhou, thanks for verifying.  I'm going to reopen this issue so that we are sure to fix the stack trace.

**Comment by rallytime** [comment]
This stacktrace is a duplicate of #26882 and #26280 and is fixed with #27681.
